models/locations.js

```diff
@@ -0,0 +1,32 @@
+module.exports = function(sequelize, DataTypes) {
+    var Locations = sequelize.define("Locations", {
```

Comment:
a typical naming convention in sequelize for tables and variables that hold the tables are as follows: in are models code we are "modeling" one item so we use the name Location e.g.:
const Location = sequelize.define("Location",{  Sequelize will automatically make the name of the table Locations, but now our model reads a little better with "db.Location"

Reply:
If you do change it, make sure to change it throughout your project.